org.gnu.emacs.json

```diff
@@ -85,8 +106,9 @@
             "buildsystem": "meson",
             "sources": [
                 {
-                    "type": "dir",
-                    "path": "wrapper"
+                    "type": "git",
```

Comment:
Not sure why this had to be included, at least to me it seems that our builds still work with the previous mechanism.

Sure we perhaps could still switch to this instead of a submodule, but I think it would be good to ship this separately, or just omit this for now if the build still works.

Reply:
That's sensible. Let me try to build it with the submodule, and I'll report back.

Reply:
Okay, I've confirmed that building with the submodule (instead of a specified commit) currently works. There is a benefit to pinning to a specific, known-to-be-working state of the git repository, but I'm not going to insist that we use one approach over the other. As both work, I leave it to your judgment!